### DISCUSSION TITLE
What is essentials.thunder?

### DISCUSSION THREAD

**Discussion by ProblemsSender** [EssentialsX/Essentials#4301] [Q&A] [answered]
What's the point of using `/thunder <true/false> <seconds>`? Did we already have `/weather`?
It output with `You enabled thunder in your world for 5 seconds.`

I don't know what is it as the EssentialsX Info does not say what it is.

**Comment by pop4959** [comment, marked as answer]
It's a bit confusing because Essentials /weather has options sun and storm, but "storm" just means [raining](https://minecraft.fandom.com/wiki/Rain).

The /thunder command enables a [thunderstorm](https://minecraft.fandom.com/wiki/Thunderstorm), which is what you would probably consider actual stormy weather.

**Reply by ProblemsSender** [reply]
hmmm, has been investigated but it seems this command and permission are not so useful. Should've explained in a little more detail how we can use it more efficiently.
